code/modules/surgery/robotics.dm

```diff
@@ -197,7 +197,7 @@
 		"<span class='warning'>You cause a short circuit in [target]'s [affected.name]!</span>")
 		target.apply_damage(rand(5,10), BURN, affected)
 
-/datum/surgery_step/robotics/fix_organ_robotic //For artificial organs
+/datum/surgery_step/robotics/fix_ORGAN_ROBOT //For artificial organs
```

Comment:
Why did you rename these? Typepaths should not have capitals.

Reply:
Ahahaha oops this was uh. A find-and-replace to fix the thing that made it not compile.